### PR TITLE
fix: fix error when updating logo with BaseLayout

### LINKED ISSUE #76: Error trying to update logo src when using `BaseLayout`
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.17.0

### Plugin version

1.9.2

### Node.js version

18

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

Ventura 13.14.1

### Description

When using `BaseLayout` like this:

```js
  await fastify.register(swaggerUi, {
    routePrefix: '/docs',
    theme: {
      title: 'My REST API'
    },
    uiConfig: {
      layout: 'BaseLayout'
    }
  });
```

I got JavaScript error in the console when opening `/docs`:

```
swagger-initializer.js:30 Uncaught TypeError: Cannot set properties of null (setting 'src')
    at window.onload (swagger-initializer.js:30:15)
```

Please note that the UI still works perfectly.

I believe we need to only execute the code that updates logo `src` if the logo element was found in first place.

### Steps to Reproduce

In the description.

### Expected Behavior

No JavaScript errors when using `BaseLayout`.

### PR DESCRIPTION
**Pull request by satazor**
Fixes #76 

Not sure how to test this. The current test suit checks the HTML response, meaning there's no way to test that actual runtime JavaScript errors occur.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/swagger-initializer.js
Comment (yashchaddha) [inline lib/swagger-initializer.js:1]: Looks Good to me !

File: lib/swagger-initializer.js [hunk 1]
```diff
@@ -35,7 +35,10 @@ function swaggerInitializer (opts) {
 
     if (logoData) {
       const img = document.querySelector('#swagger-ui > section > div.topbar > div > div > a > img')
-      img.src = logoData
+
+      if (img) {
+        img.src = logoData
+      }
     }
```
Comment (Uzlopak) [inline lib/swagger-initializer.js:42]:
```suggestion
    if (
      logoData &&
      resConfig.layout === "StandaloneLayout"
    ) {
      const img = document.querySelector('#swagger-ui > section > div.topbar > div > div > a > img')
      img.src = logoData
    }
```

**Review by Uzlopak — CHANGES_REQUESTED** [review]
Your code is one way to solve it. 

The issue here is, that swagger-ui supplies two layouts: 'BaseLayout' and 'StandaloneLayout'
In StandaloneLayout we know where the logo needs to be patched. In BaseLayout we dont have a logo, thats why we get that error.

We should check if we have StandaloneLayout and if so, patch it. 

Also by checkit if layout is StandaloneLayout the reported error becomes more obvious, why it never could work in BaseLayout

File: lib/swagger-initializer.js (hunk 1 above)
Comment (satazor) [reply lib/swagger-initializer.js:42]: The issue with this approach is that afaik swagger supports custom layouts, which may or may not contain a logo. Not sure if this is something this package supports, but if it does, and if I specify a custom layout with a logo, the above change wouldn’t patch the logo src

Comment (satazor) [reply lib/swagger-initializer.js:42]: If this package indeed supports custom layouts, this would actually be a breaking change

**Review by Uzlopak — APPROVED** [review]
LGTM

Thanks for your work

**Pull request merged**